ISISScientificComputing/autoreduce
# create a Docker environment to run Mantid

### Summary of work
* Added Dockerfile, and python class to control access
* Created accompanying tests
* Added documentation for manual use

### How to test your work
* Ensure tests pass on Travis
* Setup a new ansible node and follow the instruction for manual testing in the `reduction_docker/mantid/READMe.md`

### Additional comments
Note this does not include hooking up the QueueProcessors to use Mantid Docker. This will be done in a separate PR

Fixes #341
